docs/resources/msteams_recipient.md

```diff
@@ -2,6 +2,10 @@
 
 `honeycombio_msteams_recipient` allows you to define and manage an MSTeams recipient that can be used by Triggers or BurnAlerts notifications.
 
+-> **NOTE** Microsoft has deprecated Teams Incoming Webhooks.
```

Comment:
would it be clearer to say something about "connectors" here?

Reply:
Agreed! Will update.